oneview/resource_server_profile_template.go

```diff
@@ -325,90 +367,87 @@ func resourceServerProfileTemplate() *schema.Resource {
 									"port_id": {
 										Type:     schema.TypeString,
 										Optional: true,
-										DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
-											if new == "Auto" {
-												return true
-											}
-											return false
-										},
 									},
 									"requested_mbps": {
 										Type:     schema.TypeString,
 										Optional: true,
 									},
-									"allocated_mbps": {
-										Type:     schema.TypeInt,
-										Computed: true,
-									},
-									"allocated_vfs": {
-										Type:     schema.TypeInt,
-										Computed: true,
-									},
+									// "allocated_mbps": {
+									// 	Type:     schema.TypeInt,
+									// 	Computed: true,
+									// },
+									// "allocated_vfs": {
+									// 	Type:     schema.TypeInt,
+									// 	Computed: true,
+									// },
 
 									"requested_vfs": {
 										Type:     schema.TypeString,
 										Optional: true,
 									},
-									"state": {
-										Type:     schema.TypeString,
-										Computed: true,
-									},
-									"status": {
-										Type:     schema.TypeString,
-										Computed: true,
-									},
-									"wwnn": {
-										Type:     schema.TypeString,
-										Optional: true,
-									},
-									"wwpn": {
-										Type:     schema.TypeString,
-										Optional: true,
-									},
-									"wwpn_type": {
-										Type:     schema.TypeString,
-										Optional: true,
-									},
-
-									"interconnect_port": {
-										Type:     schema.TypeString,
-										Optional: true,
-									},
-									"interconnect_uri": {
-										Type:     schema.TypeString,
-										Computed: true,
-									},
-
-									"mac": {
-										Type:     schema.TypeString,
-										Optional: true,
-									},
+									//"state": {
```

Comment:
are those available during get call? if not we need to remove these commented part. I am not seeing those in rest api docs

Reply:
removed.